src/evaluation/examples/chaining_examples.py

```diff
@@ -0,0 +1,73 @@
+from .pipeline_example import PipelineExample
+
+operator_chaining_examples = [
+    PipelineExample(
+        "first join then aggregate",
+        """
+        import pandas as pd
+
+        con = "sqlite:///test.db"
+
+        df1 = pd.read_sql("SELECT attribute1 FROM table1", con)
+        df2 = pd.read_sql("SELECT * from table2", con)
+
+        result = df1.join(df2).max()
+        result # do something with result
+        """,
+        """
+        import pandas as pd
+
+        con = "sqlite:///test.db"
+
+        temp = pd.read_sql("SELECT * FROM (SELECT attribute1 FROM table1) JOIN (SELECT * from table2) LIMIT 0", con).columns
+        result = pd.read_sql(f"SELECT {', '.join(['MAX(' + c + ') AS max_' + c for c in temp])} FROM (SELECT attribute1 FROM table1) JOIN (SELECT * from table2)", con)
+        result # do something with result
+        """,
+    ),
+    PipelineExample(
```

Comment:
very nice. Thanks for writing tests. It helped me to understand you implemented :D